functions/graphql/schema/message.js

```diff
@@ -0,0 +1,94 @@
+const { gql, UserInputError, AuthenticationError } = require("apollo-server-express");
+const { Message } = require("../models");
+// We define a schema that encompasses all of the types
+// necessary for the functionality in this file.
+module.exports.schema = gql`
+  type Message {
+    id: ID
+    subject: String
+    message: String
+    recipients: [ID]! @auth(requires: [director])
+    hasRead: Boolean
+    date: Date
+  }
+
+  # We can extend other graphQL types using the "extend" keyword.
+  extend type Query {
+    message(id: ID!): Message @auth(requires: [authenticated])
+    messages: [Message]
+  }
+
+  extend type Mutation {
+    messageCreate(
+      subject: String!
+      message: String!
+      recipients: [ID]!
+    ): Rank @auth(requires: [director])
```

Comment:
This should return the Message type.
```suggestion
    ): Message @auth(requires: [director])
```